lib/ansible/cli/galaxy.py

```diff
@@ -312,10 +313,15 @@ def add_login_options(self, parser, parents=None):
                                   help='Identify with github token rather than username and password.')
 
     def add_info_options(self, parser, parents=None):
-        info_parser = parser.add_parser('info', parents=parents, help='View more details about a specific role.')
-        info_parser.set_defaults(func=self.execute_info)
+        galaxy_type = 'role'
+        metavar = 'role_name[,version]'
+        if parser.metavar == 'COLLECTION_ACTION':
+            galaxy_type = 'collection'
+            metavar = 'collection_name'
 
-        info_parser.add_argument('args', nargs='+', help='role', metavar='role_name[,version]')
+        info_parser = parser.add_parser('info', parents=parents, help='View more details about a specific {0}.'.format(galaxy_type))
+        info_parser.set_defaults(func=self.execute_info)
+        info_parser.add_argument('args', nargs='+', help='{0}'.format(galaxy_type), metavar=metavar)
```

Comment:
should it be `nargs='?'`, expecting a single collection/role? It looks like although multiple roles were accepted and looped over, they were silently ignored except the last.

Reply:
Good spot. I agree it should be '?'.

Reply:
Leaving it as '+' and now support multiple collections on the command line.